Test_Task/main.rb

```diff
@@ -0,0 +1,96 @@
+require 'watir'
+require 'nokogiri'
+require 'json'
+require 'date'
+require_relative './transactions'
+
+class WebBanking
+  attr_reader :browser, :page, :operation, :desc_operation
+  def initialize
+    @browser = Watir::Browser.new :chrome
+  end
+
+  def parse_accounts
+    sign_in
+    sleep(3)
+    accounts = browser.divs(class: %w(contract status-active))
+    info_list = accounts.map do |div|
+      Watir::Wait.until { div.div(class: "contract-cards").a.present? }
+      div.div(class: "contract-cards").a.click
+      go_to_info
+      go_to_transactions
+      data = {
+          name: parse_name,
+          balance: parse_balance,
+          currency: parse_currency,
+          nature: parse_nature,
+          transactions: get_transaction_info.map(&:to_hash)  #transaction.get_transaction_info(browser)
```

Comment:
it's ugly, you named your method `get_transaction_info`, 1 transaction, right? -yes, and after that you're using array's method `map`, wtf?

name this method `transactions`